addon/adapters/debug-adapter.js

```diff
@@ -1,8 +1,7 @@
-import { A, isArray } from '@ember/array';
-import DataAdapter from '@ember/debug/data-adapter';
+import { DebugAdapter as DataAdapter } from 'ember-data/-private';
```

Comment:
Lets maybe not do the rename as the Ember.js provided class is called DataAdapter, so this might be confusing.